examples/userapi/07_sparse_operations.ipynb

```diff
@@ -0,0 +1,617 @@
+{
```

Comment:
<p>These are nice figs, I would really like to had in the IPDPS paper</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>